Documentation/k8s-pre-reqs.md

```diff
@@ -1,5 +1,15 @@
 
-**WARNING:** Only Kubernetes v1.6.0 or higher is supported by Rook at this time (while Rook is in alpha it will track the latest release to use the latest features)
+## Minimum Version
+Kubernetes v1.6 or higher is targeted by Rook (while Rook is in alpha it will track the latest release to use the latest features). 
+
+Support is available for Kubernetes **v1.5.2**, although your mileage may vary. 
+You will need to use the yaml files from the [1.5 folder](/demo/kubernetes/1.5).
+
+## Privileges
+Creating the Rook operator requires privileges for setting up RBAC. To launch the operator you need to have created your user certificate with the `system:masters` privilege:
```

Comment:
is there a kubernetes doc link we can use here that approaches the general need for a cert?